### PR TITLE
fix: view ops correctly supported on WeightedSum View

### PR DESCRIPTION
**Pull request by henryiii**
Fix for comment in #431 by @colizz; expands NEP 13 support for WeighedSum View. Scalars like `WeighedSum()` are now supported too!

### CODE REVIEW

**Comment by henryiii** [comment]
@HDembinski and @jpivarski, would you mind checking over my PR to see if the NEP 13 code looks reasonable and the calculations are reasonable?

**Comment by jpivarski** [comment]
It looks good to me, though you're only overriding a few operations, like `+`, `-`, `*`, `/`, `//`, right? In principle, the same effect could be obtained by overriding `__add__`, `__radd__`, etc. I imagine someone might want to `np.sqrt` or `np.exp` the values in the bins, though it's unclear what this should do to the variances unless you know the derivatives of all functions. (And that might be desired as well; see #gradhep in IRIS-HEP Slack.) Perhaps unknown ufuncs (the main reason for NEP13) would apply to the values and drop the variances? Even though this is a lossy operation, I can see people wanting it in the midst of an analysis. Consider how often we write loops over bin contents, applying arbitrary functions...

Lossy operations for unrecognized ufuncs are particularly good if they're not in-place: then you still have the original variances in another object to decide what to do with them.

**Comment by henryiii** [comment]
I would rather the drop be explicit, that is, either `view.values` is used, or if there's a `.values()` method of some sort added, that be used. We can always add more functionality later, since it still throws an error if it's not in this list. But I should make this match via `dtype`, not by `__class__` (will fix).

File: tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    assert_allclose(v.variance * 2, v2.variance)
+
+    v2 = v + v[1]
+    assert_allclose(v.value + 3, v2.value)
+    assert_allclose(v.variance + 3, v2.variance)
+
+    v2 = v + bh.accumulators.WeightedSum(5, 6)
+    assert_allclose(v.value + 5, v2.value)
+    assert_allclose(v.variance + 6, v2.variance)
+
```
Comment (HDembinski) [inline tests/test_views.py:98]: Perhaps also test radd here?

Comment (henryiii) [reply tests/test_views.py:116]: Good call! Wasn't working. 😳

**Pull request merged**